bower.json

```diff
@@ -23,6 +23,8 @@
     "require-handlebars-plugin": "1.0.0",
     "aloha-editor": "oerpub/Aloha-Editor#master",
     "select2": "3.5.4",
+    "babel-polyfill": "^0.0.1",
+    "l20n": "https://github.com/tomekstach/l20n.js.git",
```

Comment:
@philschatz : I transferred the `l20n.js` into the `katalysteducation` and also I changed this file, because I like the order :)